firestore-counter/extension.yaml

```diff
@@ -107,4 +122,8 @@ params:
       What is the path to the document where the extension can keep its internal state?
     default: _firebase_ext_/sharded_counter
     example: _firebase_ext_/sharded_counter
+    validationRegex: "^[^/]+/[^/]+(/[^/]+/[^/]+)*$"
+    validationErrorMessage:
+      Please enter a document path, not a collection path. It should have an even
```

Comment:
_(If I make this change in the file directly, we'll lose the comment above)_

Enter a document path, not a collection path. The path must have an even number of segments, for example: `my_collection/doc` or `my_collection/doc/subcollection/doc`, but not `my_collection`.